closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -185,7 +188,8 @@ public void run() {
     }
 
     /* Native digest interfaces */
-    private static final native int loadCrypto(boolean traceEnabled);
+
```

Comment:
Please add the other blank lines so this is identical to newer versions.

Reply:
added

Reply:
There's no need for this file to be different than newer versions, e.g. https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java.

Reply:
Hi Keith, I think it is already same as other versions.

Reply:
Except for several blank lines missing here:
* https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java#L215
* https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java#L244
* https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java#L272